### DISCUSSION TITLE
update dependency?

### DISCUSSION THREAD

**Discussion by tyoc213** [wasm-bindgen/wasm-bindgen#3592] [Q&A]
### Summary

cloning https://github.com/Twinklebear/wgpu-rs-test and then installing tool chain for stable rustc 1.51.0 (2fd73fabe 2021-03-23)

I get this output
```
 wasm-bindgen --out-dir target/generated/ --web target/wasm32-unknown-unknown/debug/wgpu-test.wasm
error: 

it looks like the Rust project used to create this wasm file was linked against
version of wasm-bindgen that uses a different bindgen format than this binary:

  rust wasm file schema version: 0.2.69
     this binary schema version: 0.2.74

Currently the bindgen format is unstable enough that these two schema versions
must exactly match. You can accomplish this by either updating the wasm-bindgen
dependency or this binary.

You should be able to update the wasm-bindgen dependency with:

    cargo update -p wasm-bindgen

or you can update the binary with

    cargo install -f wasm-bindgen-cli

if this warning fails to go away though and you're not sure what to do feel free
to open an issue at https://github.com/rustwasm/wasm-bindgen/issues!

```

So here Im, none of the 2 commands didn't help

**Comment by alexcrichton** [comment]
This shows a mismatch in versions, but if neither command helps then unfortunately I'm not sure how to help. I would recommend installing the corresponding CLI version if you don't know how to update the rust dependency.

**Comment by DougLau-MNIT** [comment]
Why aren't changes to the bindgen format treated as semver-incompatible API changes?  I don't know if that would solve any problems, but it would be more logical.  How can 0.2.69 and 0.2.71 not be compatible?

**Comment by gzzo** [comment]
This worked for me: `cargo install -f wasm-bindgen-cli --version 0.2.69`

**Comment by roboteng** [comment]
I've got the same issue, but with

```
rust wasm file schema version: 0.2.87
this binary schema version: 0.2.84
``` 

I've done `cargo install -f wasm-bindgen-cli --version 0.2.87` and I'm still getting the messages saying I'm on `0.2.84`

I've also tried specifying the version in my `cargo.toml`, but that doesn't seem to change anything. I'm unsure where this dependency is coming into my project.

**Reply by daxpedda** [reply]
Happy to take a look if you have something I can take a look at!
